root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -263,8 +263,16 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
                             if (gid === target.gid) {
                                 _.each(entity.displayableRelationships(vm)(), function (r) {
                                     switch (relationshipFilter) {
-                                      case 'same-entity-types': if (r.entityTypes !== relationship.entityTypes) { return; }; break;
-                                      case 'same-relationship-type': if (r.linkTypeID() !== relationship.linkTypeID()) { return; }; break;
+                                      case 'same-entity-types':
+                                          if (r.entityTypes !== relationship.entityTypes) {
+                                              return;
+                                          }
+                                      break;
```

Comment:
I guess this break (and the one under it) should be indented with the if block, not the case? :)